src/test/java/de/redsix/pdfcompare/IntegrationTest.java

```diff
@@ -209,6 +210,27 @@ public void identicalFilenamesAreEqual() throws IOException {
         writeAndCompare(result);
     }
 
+    @Test
+    public void compareWithBase64() throws IOException {
+        String expectedBase64 = Base64.getEncoder().encodeToString(toByteArray(r("expected.pdf")));
+        String actualBase64 = Base64.getEncoder().encodeToString(toByteArray(r("actual.pdf")));
+        final CompareResult result = new PdfComparator(expectedBase64, actualBase64).compare();
+        assertThat(result.isEqual(), is(true));
```

Comment:
this should be false, since the expected.pdf and the actual.pdf files are note equal

Reply:
Oh yes, sorry...